lib/webapp.js

```diff
@@ -305,7 +305,7 @@ function getAttrib(object, path){
 
 function setAttrib(object, path, value){
   if(object === undefined || path === undefined || 
-      typeof path === 'array' && path.length === 0 || typeof object !== 'object'){
+      Array.isArray(path) && path.length === 0 || typeof object !== 'object'){
```

Comment:
The result of this 'typeof' expression was compared to 'array', but the two could never be equal.